specification/documentation/APIDescription.md

```diff
@@ -1,15 +1,15 @@
 ## Overview
 
-Use this API to send messages to citizens via email, SMS, the NHS App or letter.
+Use this API to send messages to citizens via NHS App, email, text message or letter.
```

Comment:
I wonder if we should put "text message (SMS)" to match how we list the channels on the Features page of the service catalogue? https://digital.nhs.uk/services/nhs-notify/features#:~:text=emails-,text%20messages%20(SMS),-letters

Reply:
As discussed with Jake, leaving as-is.